project-setup/templates/CONTRIBUTING-template.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to ____
```

Comment:
This is great, but surely there must be an open-source template that we can link to?  Nothing here looks unique to InnerSource.

Reply:
I haven't looked for contributing templates. For READMEs I usually forward people to 

https://m.dotdev.co/how-to-write-a-readme-that-rocks-bc29f279611a

I wonder if any of the bigger foundations have templates for these things that we can link to.

A ton of wisdom on how to run an open source project is also in https://producingoss.com/en/producingoss.html 

There's also the following from GitHub, which if you click through leads to a more detailed explanation of what belongs into a contributing file.

I'm sure there are others that I'm not aware of. A lot of the things I've written down come from practical experience in open source projects.

Reply:
I have added links to these sites to the pattern itself - also added that often it's a good idea to check what open source projects include in their docs.

Reply:
I had a bunch of links researched when I wrote mine, maybe I'll find some notes or bookmarks of that.